### ISSUE TITLE
A related work - No Normalization

### ISSUE THREAD

**Issue by hyperevolnet** [jiachenzhu/DyT#2]
As we know, normalization usually includes three parts: Momentum, Affine params (γ, β), and standardization. LN and GN don't have a Momentum term.

Refs: 
https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html
https://pytorch.org/docs/stable/generated/torch.nn.InstanceNorm2d.html
https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html
https://pytorch.org/docs/stable/generated/torch.nn.GroupNorm.html

**The proposed DyT still retains Affine params. Can you provide experimental results without them?**

In addition, our previous work ([HyperZ·Z·W Operator Connects Slow-Fast Networks for Full Context
Interaction](https://arxiv.org/pdf/2401.17948)) tried to remove Momentum and Affine params from Norms. We can achieve better results when only Standardization is retained. Please refer to Figure 6 and Table 4.

**Comment by hyperevolnet** [comment]
Moreover, we recommended our HyperZZW work to Dr. Yann LeCun and Dr. Kaiming He via email in February 2024, even though they did not respond.

**Comment by jiachenzhu** [comment]
Thank you for the suggestions. As we pointed out in our original paper, our focus is on understanding the role of normalization operations (all our studies in Section 3 are conducted before the affine layers). Therefore, we have not conducted an in-depth study on affine transformations.

Our view is that removing the affine parameters still results in a normalization layer. However, if the normalization operation itself is removed, the affine parameters alone would no longer be considered part of a normalization layer. In fact, the affine transformation can sometimes be thought of as a separate layer following the normalization. As such, we currently have no plans to explore affine transformations in detail.

We also want to highlight that all our experiments with DiT were conducted without affine parameters, which is the original setup of DiT.

Finally, we appreciate you highlighting your previous work and the results showing potential improvements when only standardization is retained. We’ll take a closer look at that paper — thank you for bringing it to our attention!

**Comment by hyperevolnet** [comment]
Thank you for the response.

First, both the _pytorch code_ and the _BN paper_ consider the **affine parameters as part of normalization**, not as a separate layer.

<img width="594" alt="Image" src="https://github.com/user-attachments/assets/3f2b2820-42eb-4497-a759-80710614e023" />



Second, our previous results show that ResNet performs very poorly after removing affine parameters (85.41 vs 77.82). So I guess that transformers would be the same. Neglecting the exploration of affine parameters could potentially limit the comprehensive understanding of normalization operations and their effects on model behavior.


Third, why [DiT](https://github.com/facebookresearch/DiT/blob/main/models.py) (_no comparison results_) and our [Terminator](https://arxiv.org/pdf/2401.17948) still perform well after removing the affine parameters is a question worth exploring.
